gapis/api/vulkan/replay.go

```diff
@@ -781,3 +781,8 @@ func (a API) QueryIssues(
 	}
 	return res.([]replay.Issue), nil
 }
+
+// ExportReplayRequest returns request type for standalone replay.
+func (a API) ExportReplayRequest() replay.Request {
+	return issuesRequest{}
```

Comment:
We really should have a new type of request, instead of using `issuesRequest`, eventually. And using `issuesRequest` won't have any framebuffer data postback, so can be hard to tell if the replay is correct or not.